### DISCUSSION TITLE
How to choose what to put for "Keyfile for initial user"?

### DISCUSSION THREAD

**Discussion by NathanWailes** [dokku/dokku#7098] [Q&A] [answered]
I'm upgrading Dokku and it's asking me a series of questions that I don't know how to answer.

The one I'm on now is "Keyfile for initial user" and the default value is `/root/.ssh/id_rsa.pub`.

**Comment by josegonzalez** [comment, marked as answer]
It should be a path to your public ssh key (if you have one). The default can be left there and it shouldn't break anything if it doesn't exist.
